src/curobo/rollout/arm_reacher.py

```diff
@@ -370,7 +370,7 @@ def convergence_fn(
                         current_pos = link_poses[k].position
                         current_quat = link_poses[k].quaternion
 
-                        pose_err, pos_err, quat_err = current_fn.forward_out_distance(
+                        pose_err, quat_err, pos_err = current_fn.forward_out_distance(
```

Comment:
so quat should be returned first before pos err??

Reply:
Yeah this bug tricked me into a 3 hour debugging session...